target/linux/realtek/files-5.10/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -4875,6 +4876,7 @@ static struct phy_driver rtl83xx_phy_driver[] = {
 		.features	= PHY_GBIT_FEATURES,
 		.flags		= PHY_HAS_REALTEK_PAGES,
 		.probe		= rtl8218d_phy_probe,
+		.config_init	= rtl9300_configure_8218d,
```

Comment:
Note, that within this patch, this does not yet exist, so probably comes from one of your other patches?